code/go/0chain.net/core/encryption/bls0chain.go

```diff
@@ -239,6 +239,17 @@ func (b0 *BLS0ChainScheme) GetSignature(signature string) (*bls.Sign, error) {
 	return &sign, nil
 }
 
+// GetClientID - get client id by public key
+func (b0 *BLS0ChainScheme) GetClientID(publicKey string) (string, error) {
+	pk := &bls.PublicKey{}
+	err := pk.DeserializeHexStr(publicKey)
+	if err != nil {
+		return "", err
+	}
+
+	return encryption.Hash(pk.Serialize()), nil
```

Comment:
ClientID is `encryption.Hash(publicKeyBytes)`, we don't need to implement it on `Scheme`, call https://github.com/0chain/0chain/blob/3ab1f87a4fa6eb0df2cc36f5681fbc458d3bf3b7/code/go/0chain.net/core/encryption/signature_scheme.go#L177 instead. So please remove the `GetClientID()` method from the scheme interface.

Reply:
fixed